cms/models.py

```diff
@@ -40,10 +43,27 @@ class Meta:
         StreamFieldPanel("content"),
     ]
 
+    subpage_types = [
+        "FacultyMembersPage",
+    ]
+
     def get_context(self, request, *args, **kwargs):
-        context = super(BootcampPage, self).get_context(request)
+        context = super().get_context(request)
         context["title"] = self.title
+        context["faculty"] = self.faculty.__dict__,
+
         return context
+    
+    def _get_child_page_of_type(self, cls):
+        """Gets the first child page of the given type if it exists"""
+        child = self.get_children().type(cls).live().first()
+        return child.specific if child else None
+
+    @property
+    def faculty(self):
+        """Gets the faculty carousel page"""
```

Comment:
Please update the docstring, it seems like copying from xPRO you forgot to update the docstring.